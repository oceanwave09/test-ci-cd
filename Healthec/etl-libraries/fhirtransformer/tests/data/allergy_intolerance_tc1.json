{
  "source_id": "8576713",
  "clinical_status": "active",
  "verification_status": "confirmed",
  "category": "medication",
  "criticality": "high",
  "code": "7980",
  "code_display": "Penicillin G",
  "code_system": "http://www.nlm.nih.gov/research/umls/rxnorm",
  "recorded_date_time": "2010-03-01",
  "practitioner_id": "2bb088a4-5259-47b2-8d0b-8d67d4af8fca",
  "patient_id": "dda84174-22dc-4af5-a5f0-2e9f828365c8",
  "onset_start_date": "2015-01-17T16:00:00+10:00",
  "onset_end_date": "2015-01-17T16:30:00+10:00",
  "manifestation_code": "247472004",
  "manifestation_display": "Hives",
  "manifestation_system": "http://snomed.info/sct"
}