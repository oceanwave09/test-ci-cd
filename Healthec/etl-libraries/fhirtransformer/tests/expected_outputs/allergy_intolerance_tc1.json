{
  "resourceType": "AllergyIntolerance",
  "identifier": [
    {
      "type": {
        "coding": [
          {
            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
            "code": "RI",
            "display": "Resource Identifier"
          }
        ]
      },
      "value": "8576713"
    }
  ],
  "clinicalStatus": {
    "coding": [
      {
        "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
        "code": "active"
      }
    ]
  },
  "verificationStatus": {
    "coding": [
      {
        "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-verification",
        "code": "confirmed"
      }
    ]
  },
  "category": [
    "medication"
  ],
  "criticality": "high",
  "code": {
    "coding": [
      {
        "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
        "code": "7980",
        "display": "Penicillin G"
      }
    ]
  },
  "patient": {
    "reference": "Patient/dda84174-22dc-4af5-a5f0-2e9f828365c8"
  },
  "recordedDate": "2010-03-01",
  "recorder": {
    "reference": "Practitioner/2bb088a4-5259-47b2-8d0b-8d67d4af8fca"
  },
  "reaction": [
    {
      "manifestation": [
        {
          "coding": [
            {
              "system": "http://snomed.info/sct",
              "code": "247472004",
              "display": "Hives"
            }
          ]
        }
      ]
    }
  ]
}